srcpkgs/python3-ecdsa/template

```diff
@@ -1,23 +1,29 @@
 # Template file for 'python3-ecdsa'
 pkgname=python3-ecdsa
-version=0.14.1
-revision=5
-wrksrc="ecdsa-${version}"
+version=0.16.1
+revision=1
+wrksrc="python-ecdsa-python-ecdsa-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-six"
+checkdepends="python3-pytest python3-hypothesis python3-coverage"
 short_desc="Implementation of ECDSA in Python3"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/warner/python-ecdsa/"
-distfiles="${PYPI_SITE}/e/ecdsa/ecdsa-${version}.tar.gz"
-checksum=64c613005f13efec6541bb0a33290d0d03c27abab5f15fbab20fb0ee162bdd8e
+distfiles="https://github.com/warner/python-ecdsa/archive/python-ecdsa-${version}.tar.gz"
+checksum=3b9fb3cc04d7375e1087dc5b31d6e3edd76a07b307f417b78b8924e164438828
 
 pre_build() {
 	# use system six
 	find . -type f -name '*.py' -exec sed -i 's/from \(ecdsa\|\)\.six/from six/g' {} +
 	rm -f ecdsa/six.py
 }
```

Comment:
You can probably remove this block, given https://github.com/warner/python-ecdsa/releases/tag/python-ecdsa-0.14.1